octodns/manager.py

```diff
@@ -500,7 +501,7 @@ def _populate_and_plan(
         if desired:
             # This is an alias zone, rather than populate it we'll copy the
             # records over from `desired`.
-            for _, records in desired._records.items():
+            for _, records in desired._pre_processing_records.items():
```

Comment:
I believe this would break things that depend on/expect the processors to have run. E.g. if you're using filtering processors to remove/ignore things the alias zone would have them and the parent wouldn't.